assets/src/domain/blocks/event-attendees/controls/SelectDatetime.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+import useDatetimes from '../hooks/useDatetimes';
+
+const SelectDatetime: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
```

Comment:
let's move this into a new folder named something like:

```
assets/src/domain/blocks/components/entitySelectors/
```

because a dropdown control for selecting datetimes will undoubtedly be required in other blocks

Reply:
no comment or action on this?